ui/app/styles/components/actions.scss

```diff
@@ -36,6 +36,9 @@
       padding-left: 0;
       padding-right: 0;
       border-width: 0;
+      span {
+        text-align: left;
+      }
```

Comment:
I assume it's fine, but out of curiosity, is this directly related to the dep updates?

And is nothing else needed?  I might expect more breakage from a major version bump.

Reply:
PR description updated after I posted this.  thanks!  :+1: